packages/patternfly-3/patternfly-react-extensions/src/components/TableGrid/TableGridHead.js

```diff
@@ -1,30 +1,115 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import { Grid } from 'patternfly-react';
+import { Button, Grid, Icon, OverlayTrigger, noop } from 'patternfly-react';
+import { Tooltip } from '../../../../patternfly-react/src/components/Tooltip';
 
 /**
  * TableGridHead Component for PatternFly
  */
 
-const TableGridHead = ({ children, className, ...props }) => {
+const TableGridHead = ({
+  id,
+  children,
+  className,
+  showCheckbox,
+  allSelected,
+  partialSelected,
+  onToggleSelection,
+  toggleTip,
+  allSelectedAriaLabel,
+  partialSelectedAriaLabel,
+  noneSelectedAriaLabel,
+  ...props
+}) => {
   const classes = classNames('table-grid-pf-head', className);
+
+  const getTipText = () => {
+    if (toggleTip) {
+      return toggleTip;
+    }
+    if (allSelected) {
+      return 'All Selected';
```

Comment:
can we provided these strings as props or allow override of the `getTipText` function used? (i18n)

Reply:
The toggleTip is for that purpose, i provided one instead of all variations.